themes/bootstrap5/templates/RecordDriver/ProQuestFSG/data-publicationDetails.phtml

```diff
@@ -0,0 +1,14 @@
+<?php foreach ($data as $field): ?>
```

Comment:
I've implemented @sturkel89 's feedback to add the MARC 773 info:
> Publication information display in item records could be improved

I did not yet do what was suggested to display either the 773 info OR the 260 as a backup -- they are both being displayed.  The publisher field ("Kenyon College" here) is unique to the 260.  
The publication date (Winter 1992 here) does seem to be duplicated between the two, so in theory I could skip that on the Published line.  But I'm not sure how opinionated to be about that.

![image](https://github.com/user-attachments/assets/8ad9791a-698a-4e56-aa7f-501cfa2b9e57)

Reply:
While it's a little ugly to repeat the information, I think the 260 and 773 fields are semantically different and won't necessarily always include redundant information. I'd be inclined to continue displaying both in the interest of completeness.

Reply:
I think I agree.  Could we change the heading "Published" though to "Published by" to distinguish from "Published in"?  I know the date part is not technically "by" but I think it could still work.

Reply:
Not a bad idea, just a question of how to implement it in the most translation-friendly way (i.e. do we revise existing keys or create a new key? Is the existing key used in multiple contexts that might have different semantics?)

Reply:
I think it would have to be a new key; I see 'Published' used for two different meanings in RecordDataFormatterFactory alone, referencing getPublicationDetails and getDateSpan.

All that said, I think this would be a separate PR as it affects other backends and others might want to weigh in.

Reply:
I definitely agree, it's out of scope for this PR but worth looking at separately.

Reply:
This all makes sense and I appreciate the need to consider language strings/translation. I am going to post follow-up comments with other suggestions for tweaking results browse and record view after I do some more analysis of PQ content and fields.